__tests__/recaptcha-test.js

```diff
@@ -1,13 +1,182 @@
-jest.dontMock('../src/index');
+// Mock grecaptcha, see below
+const grecaptchaMock = {
+	reset: function() {
+		return true
+	},
+	render: function() {
+		return true
+	}
+}
 
+// JSDOM configuration (should be done before requiring anything else, see: http://airbnb.io/enzyme/docs/guides/jsdom.html)
+const {JSDOM} = require('jsdom');
+const jsdom = new JSDOM('<!doctype html><html><body></body></html>');
+const {window} = jsdom;
+// set grecaptchaMock as a property on JSDOM window, will now be accessable by component when rendered in tests
+window.grecaptcha = grecaptchaMock
+
+function copyProps(src, target) {
+	const props = Object.getOwnPropertyNames(src).filter(prop => typeof target[prop] === 'undefined').reduce((result, prop) => ({
+		...result,
+		[prop]: Object.getOwnPropertyDescriptor(src, prop)
+	}), {});
+	Object.defineProperties(target, props);
+}
+
+global.window = window;
+global.document = window.document;
+global.navigator = {
+	userAgent: 'node.js'
+};
+copyProps(window, global);
+
+// Import React, test utils, component
 import React from 'react';
 import ReactTestUtils from 'react-dom/test-utils';
 import Recaptcha from '../src/index';
+import sinon from 'sinon';
+
+// Lodash for deep equality testing
+import _ from 'lodash'
+
+// Import enzyme and configure for React 15
+import Adapter from 'enzyme-adapter-react-15';
+import {configure, render, mount} from 'enzyme';
+configure({adapter: new Adapter()});
+
+// Test method for test props
+function testFunction() {
+	return true
+}
+
+// Test props with all props specified
+const testProps = {
+	className: 'testClass',
+	onloadCallbackName: 'testCallBackName',
+	elementID: 'testElementID',
+	onloadCallback: testFunction,
+	verifyCallback: testFunction,
+	expiredCallback: testFunction,
+	render: 'testRender',
+	sitekey: 'testSitekey',
+	theme: 'testTheme',
+	type: 'testType',
+	verifyCallbackName: 'testVerifyCallbackName',
+	expiredCallbackName: 'testExpiredCallbackName',
+	size: 'testSize',
+	tabindex: 'testTabindex',
+	hl: 'testHl',
+	badge: 'testBadge'
+};
```

Comment:
coding style? Is indent two space?

Reply:
Looks like indenting is tabs right now, should I change it to two spaces?

Reply:
See [e7ed11d](https://github.com/appleboy/react-recaptcha/pull/223/commits/e7ed11dcb8a589ca93a8d6a7f5a65246bd2cc8e6)